tests/integration-tests/tests/createami/test_createami/test_build_image_custom_components/image.config.yaml

```diff
@@ -2,12 +2,7 @@ Build:
   InstanceType: {{ instance_type }}
   ParentImage: {{ parent_image }}
   Components:
-    # Test arn custom component in eu-west-1 except Rocky and Ubuntu because stig-build-linux-high component doesn't support these ParallelCuster AMIs.
-    # Test script custom component in other regions.
-    {% if region == "eu-west-1" and "rocky" not in os and "ubuntu" not in os %}
     - Type: arn
-      Value: arn:{{ partition }}:imagebuilder:{{ region }}:aws:component/stig-build-linux-high/2024.2.3/1
-    {% else %}
+      Value: arn:{{ partition }}:imagebuilder:{{ region }}:aws:component/hello-world-linux/1.0.0/1
```

Comment:
is this available in all regions ?

Reply:
or the 2 regions we use in test?

Reply:
The hello-world is available in all regions